src/libraries/Common/tests/System/Net/Security/FakeNtlmServer.cs

```diff
@@ -396,6 +396,12 @@ private void ValidateAuthentication(byte[] incomingBlob)
 
         public void ResetKeys()
         {
+            if (_clientSeal != null)
```

Comment:
not sure about this one -- clearly, it still won't dispose the last set it creates. just reduces the noise. happy to revert.

Reply:
LGTM

Reply:
However, you may want to actually fix that in the product code too:
https://github.com/dotnet/runtime/blob/b9c99648783d0c28afdee394a93516c0bd228165/src/libraries/System.Net.Security/src/System/Net/NTAuthentication.Managed.cs#L775
by adding
```
            _clientSeal?.Dispose();
            _serverSeal?.Dispose();
```
(to match the exact code used in `Dispose` call)

Reply:
There's still so much noise from this codepath relative to all the test noise that it's hard to find product leaks. I think I will go ahead make it disposable.

Reply:
oh, nice catch. I probably would have found that if the tests weren't so noisy!